docs/dapps/index.md

```diff
@@ -12,11 +12,7 @@ lang: en-US
 
 # Using Ethereum {#using-ethereum}
 
-<div class="featured">
-
-**Want to get started using Ethereum?** There are hundreds of decentralized applications (dapps) built on Ethereum that you can use today.
-
-</div>
+**Want to get started using Ethereum?** There are hundreds of decentralized applications (dapps) built on Ethereum that you can use today. {.l4 .featured}
```

Comment:
This change will require updating all translation markdown pages as well. Grep for `class="featured"` & you'll see all the files (there's ~70).

Reply:
Yeah I spotted, this isn't a breaking change right now though. Happy to pick that up when continuing